physical_property/pKa/microstates/README.md

```diff
@@ -0,0 +1,10 @@
+## pKa Microstates of SAMPL7 Molecules
+
+24 small molecules are assigned moleculed IDs in the form of SMXX from SM25 to SM46. This directory includes their enumerated microstates.
+
+Please note, the microstate lists provided here may not be comprehensive.
+
+### Manifest
+- `SMXX_microstates.csv` - Files include updated microstate IDs and canonical SMILES of each microstate. The first microstate in each CSV file indicated by `SMXX_000` is the neutral reference state.
```

Comment:
Maybe "the neutral reference state" -> "our selected neutral reference state". This is the one that all predictions should be made relative to, right? 

(Same point would apply to anywhere else you mention neutral reference state.)

Reply:
Updated.